application/views/admin/surveysgroups/update.php

```diff
@@ -45,7 +45,14 @@
             <div id="templateSettingsFortThisGroup" class="tab-pane">
                 <?php
                     if (is_a($templateOptionsModel, 'TemplateConfiguration')){
-                        Yii::app()->getController()->renderPartial('/themeOptions/surveythemelist', array( 'oSurveyTheme'=> $templateOptionsModel, 'pageSize'=>$pageSize ));
+                        Yii::app()->getController()->renderPartial(
+                            '/themeOptions/surveythemelist',
+                            array(
+                                'oSurveyTheme' => $templateOptionsModel,
+                                'pageSize' => $pageSize,
+                                'SurveysGroup' => $model
```

Comment:
What is the idea behind to put SurveysGroup additionally?

Reply:
**This is the fix in fact**

Reply:
ok, thank you for clarification